server/models/schema.sql

```diff
@@ -1564,6 +1564,8 @@ CREATE TABLE `invoice` (
   `description`         TEXT NOT NULL,
   `created_at`          TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `is_distributable`    TINYINT(1) NOT NULL ,
+  `is_credit_note`      TINYINT(1),
+  `credit_note_by`      SMALLINT(5) UNSIGNED,
```

Comment:
Just because / # / invoices draws data directly from the Invoices table, which is why I added the column and finally is_credit_note credit_note_by to avoid queries joins with the posting log table but also of general_ledger in the case or posting journal is empty